public/apps/account/account-nav-button.tsx

```diff
@@ -111,36 +112,45 @@ export function AccountNavButton(props: {
         View roles and identities
       </EuiButtonEmpty>
       {horizontalRule}
-      <EuiButtonEmpty data-test-subj="switch-tenants" size="xs" onClick={showTenantSwitchPanel}>
-        Switch tenants
-      </EuiButtonEmpty>
+      <EuiFlexItem grow={false}>
+        <EuiButtonEmpty flush="left" data-test-subj="switch-tenants" size="xs" onClick={showTenantSwitchPanel}>
+          Switch tenants
+        </EuiButtonEmpty>
+      </EuiFlexItem>
       {props.isInternalUser && (
         <>
           {horizontalRule}
-          <EuiButtonEmpty
-            data-test-subj="reset-password"
-            size="xs"
-            onClick={() =>
-              setModal(
-                <PasswordResetPanel
-                  coreStart={props.coreStart}
-                  username={props.username}
-                  handleClose={() => setModal(null)}
-                  logoutUrl={props.config.auth.logout_url}
-                />
-              )
-            }
-          >
-            Reset password
-          </EuiButtonEmpty>
+        <EuiFlexItem grow={true}>
+            <EuiButtonEmpty
+              data-test-subj="reset-password"
+              size="xs"
+              onClick={() =>
+                setModal(
+                  <PasswordResetPanel
+                    coreStart={props.coreStart}
+                    username={props.username}
+                    handleClose={() => setModal(null)}
+                    logoutUrl={props.config.auth.logout_url}
+                  />
+                )
+              }
+            >
+              Reset password
+            </EuiButtonEmpty>
+          </EuiFlexItem>
         </>
       )}
-      <LogoutButton
-        authType={props.config.auth.type}
-        http={props.coreStart.http}
-        divider={horizontalRule}
-        logoutUrl={props.config.auth.logout_url}
-      />
+      {horizontalRule}
+      <EuiFlexItem grow={true}>
+        <EuiButtonEmpty>
+          <LogoutButton 
+            authType={props.config.auth.type}
+            http={props.coreStart.http}
+            divider={null}
```

Comment:
> Please preserve the original whitespace when possible, it looks like there was some changes that I wasn't clear about that might need more documentation before we could merge this change

Sorry I am not sure what you mean about the original whitespace? In the code or in the PR documentation? 





> What about removing the `divider` parameter entirely?

I originally removed the `divider` field but the LogoutButton requires it as a parameter; it also is slightly different looking then the other dividers being used since it divides as part of the Logout Button versus the menu.

Reply:
> LogoutButton requires it as a parameter

Can we remove that requirement if we aren't going to use it in all cases?